backend/src/Data/AccessRights.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Database.Data;
+
+public class AccessRights
+    : Entity
+{
+    public Guid InstitutionId { get; set; }
+    public int AllowedUserCount { get; set; }
+    public List<Guid> UserOfInstitution { get; set; }
```

Comment:
Could you make the name of this field more telling by adding some adjective? (above it is allowed)